### ISSUE TITLE
Get Other Plugins data

### ISSUE THREAD

**Issue by dhananjay1995** [BuildFire/sdk#336]
i want to navigate to other plugin present in same folder, 
how can i get all the details like its pluginId, InstanceId,etc

**Comment by DanielHindi** [comment]
Can you please clarify on what you mean by the "same folder". If you are referring to your test environment that is he relevant to the app. When developing plug-ins each plug-in is autonomous. Each of your plug-ins may or may not be deployed to the same application that being said if you have full control over where your plug-ins are going to be deployed in a particular app and you're making an assumption  that both plug-ins are in the same app that still doesn't tell you which plug in instance you're referring to. Think of a plug-in as a class and an instance as an object of that class. Each plug-in can get its IDs both plug-in type and instance ID by calling buildfire.getContext()
However if you want to look up other plug-ins in the same app usually buildfire plugin instances library

https://github.com/BuildFire/sdk/wiki/Plugin-Instances

To see it in action look at any launcher plugin like Folder Plugin
https://github.com/BuildFire/folderPlugin

 Basically in the control side of your plug-in a pop-up comes up with a search for other plug-ins within this application once you to select which ones they want the pop-up will return back to you it's IDs you can and save that and use that for navigation.

One thing to keep in mind is if you are referencing plug-in titles and icons  your reference may grow stale because as the user changes it in the application your reference may not be updated. To fix this issue make sure you use buildfire dynamic data 
https://github.com/BuildFire/sdk/wiki/How-to-use-the-Datastore-Dynamic-Data

 You can also see this in action in the folder plug-in what it basically does as it passes the IDs to dynamic data and the library returns back the latest title and Icon. 

Hope this helps

**Comment by NancyHaddad** [comment]
Closing due to inactivity
